src/runtime_src/core/edge/user/device_linux.cpp

```diff
@@ -30,9 +31,159 @@ namespace {
 
 namespace query = xrt_core::query;
 using key_type = query::key_type;
+xclDeviceHandle handle;
+xclDeviceInfo2 deviceInfo;
 
 static std::map<query::key_type, std::unique_ptr<query::request>> query_tbl;
 
+static zynq_device*
+get_edgedev(const xrt_core::device* device)
+{
+  return zynq_device::get_dev();
+}
+
+struct devInfo
+{
+   static boost::any
+    get(const xrt_core::device* device,key_type key)
+  {
+    auto edev = get_edgedev(device);
+    switch (key)
+    {
+    case key_type::edge_vendor:
+      return deviceInfo.mVendorId;
+    case key_type::rom_vbnv:
+      return std::string(deviceInfo.mName);
+    case key_type::rom_ddr_bank_size:
+      return (deviceInfo.mDDRSize >> 30);
```

Comment:
Magic numbers.  Could you define what is happening here.

Reply:
As stated the value expected is in GBytes and I am reading the value in bytes, converted it to expected value.

Reply:
Please remove the magic number and comment this code.
For example:
```
case key_type::rom_ddr_bank_size:
{ 
  static const uint32_t BYTES_TO_GBYTES = 30;
  return (deviceInfo.mDDRSize >> BYTES_TO_GBYTES);
}
```